src/app/pages/Sandbox/Editor/Workspace/Project/index.js

```diff
@@ -155,26 +147,23 @@ export default class Project extends React.PureComponent {
               </Link>
             </Item>
           </div>}
-        <WorkspaceSubtitle>Statistics</WorkspaceSubtitle>
-        <ViewCountContainer>
-          <ViewCount>{likeCount}{' '}</ViewCount>
-          <ViewCountDescription>
-            {likeCount === 1 ? ' like' : ' likes'}
-          </ViewCountDescription>
-        </ViewCountContainer>
-        <ViewCountContainer>
-          <ViewCount>{forkCount}{' '}</ViewCount>
-          <ViewCountDescription>
-            {forkCount === 1 ? ' fork' : ' forks'}
-          </ViewCountDescription>
-        </ViewCountContainer>
-        <ViewCountContainer>
-          <ViewCount>{viewCount}{' '}</ViewCount>
-          <ViewCountDescription>
-            unique
-            {viewCount === 1 ? ' view' : ' views'}
-          </ViewCountDescription>
-        </ViewCountContainer>
+
+        {githubUrl &&
```

Comment:
if `githubUrl` is an empty string (i.e. not a `null` or `undefined`), this will insert an HTML comment into the DOM. Not a big deal, but if you care, change the line to:
```js
{!!githubUrl &&
```